src/internals/tooltip.js

```diff
@@ -246,18 +246,70 @@ extend(ChartInternal.prototype, {
 		const tWidth = $$.tooltip.property("offsetWidth");
 		const tHeight = $$.tooltip.property("offsetHeight");
 		const position = positionFunction.call(this, dataToShow, tWidth, tHeight, element);
+		const x = selectedData[0].x || undefined;
 
 		// Set tooltip
 		$$.tooltip
 			.style("top", `${position.top}px`)
 			.style("left", `${position.left}px`);
+
+		if (config.tooltip_linked) {
+			$$.api.internal.charts.forEach(c => {
+				const isInDom = document.body.contains(c.element);
+
+				if (c !== $$.api && isInDom) {
+					const tt = c.internal.tooltip;
+
+					if (tt.style("display") !== "block") {
+						c.tooltip.show({x: x});
+					}
+				}
+			});
+		}
+
+		if (config.tooltip_onShown) {
```

Comment:
There're already undocumented tooltip.onshow/onhide options.
It'll be good using them instead.

Reply:
I saw these, but they don't functioning in  (what I consider) the expected manner. The current implementation is called in the show() method of api.tooltip.js. It fires once when the tooltip is originally shown and then never again. If the tooltips are linked, they will fire all the time because the show method is called explicitly in the linked tooltip handler.

I'll look at the implementation some more today.